### DISCUSSION TITLE
Configure default value settings

### DISCUSSION THREAD

**Discussion by zh199609** [seleniumbase/SeleniumBase#2555] [Q&A] [answered]
How can I overwrite the default values in the settings. py file in the source code in the project

**Comment by mdmintz** [comment, marked as answer]
You can pass a custom settings file to overwrite those values: [examples/custom_settings.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/custom_settings.py).

Eg, with `pytest` tests:

```bash
pytest --settings=custom_settings.py
```

There's also an arg in the `SB()` format:

```python
with SB(settings_file="custom_settings.py") as sb:
```

If you're using the `Driver()` format, set settings individually before launching the browser. Eg:

```python
from seleniumbase import Driver
from seleniumbase.config import settings

settings.SMALL_TIMEOUT = 7
settings.LARGE_TIMEOUT = 10

driver = Driver()
# ...
driver.quit()
```
